specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1309 @@
+          "x-ms-client-flatten": true
+        },
+        "location": {
+          "description": "The location of the resource.",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "tags": {
+          "description": "The resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "name": {
+          "description": "The name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "The ID of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PeeringSku": {
+      "description": "The SKU that defines the tier and kind of the peering.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the peering SKU.",
+          "enum": [
+            "Basic_Exchange_Free",
+            "Basic_Direct_Free",
+            "Premium_Direct_Free",
+            "Premium_Exchange_Metered"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "name",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "description": "The tier of the peering SKU.",
+          "enum": [
+            "Basic",
+            "Premium"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "tier",
+            "modelAsString": true
+          }
+        },
+        "family": {
+          "description": "The family of the peering SKU.",
+          "enum": [
+            "Direct",
+            "Exchange"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "family",
+            "modelAsString": true
+          }
+        },
+        "size": {
+          "description": "The size of the peering SKU.",
+          "enum": [
+            "Free",
+            "Metered",
+            "Unlimited"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "size",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PeeringProperties": {
+      "description": "The properties that define connectivity to the Microsoft Cloud Edge.",
+      "type": "object",
+      "properties": {
+        "direct": {
+          "$ref": "#/definitions/PeeringPropertiesDirect",
+          "description": "The properties that define a direct peering."
+        },
+        "exchange": {
+          "$ref": "#/definitions/PeeringPropertiesExchange",
+          "description": "The properties that define an exchange peering."
+        },
+        "peeringLocation": {
```

Comment:
>peeringLocation [](start = 9, length = 15)

<same comment>

Is this the peering location itself (ex: Redmond) or peeringLocation resource name ? (peeringLocationFoo that represents Redmond,WA). 
I would consider treating it as a peering location resource uri.
For ex: names can clash.. just peeringLocation=Redmond is ambiguous it can be present in multiple states/countries)
Does that make sense ?

Reply:
@sanjaiganesh The peeringLocation is a proxy resource, but yes it is the name (Redmond). They are unique and represent the edge locations we have in Azure. It is also unique based on the facility ID which is in the connection, we have a check to ensure the ID and location match.

Reply:
The peeringLocation resource "name" and  the "peeringLocation" itself have the same value.